### PR TITLE
[web] WebBrowser.openBrowserAsync

### PR DESCRIPTION
**Pull request by EvanBacon**
# How

`WebBrowser.openBrowserAsync` will link to another window with a blank target on web.

### CODE REVIEW

File: packages/expo-web-browser/src/ExpoWebBrowser.web.ts [hunk 1]
```diff
@@ -1,5 +1,15 @@
...
 export default {
   get name() {
     return 'ExpoWebBrowser';
   },
+  async openBrowserAsync(
+    url: string,
+    browserParams: OpenBrowserParams = {}
+  ): Promise<BrowserResult> {
+    const { target = ' ', features, replace } = browserParams;
+    window.open(url, target, features, replace);
```
Comment (ide) [inline packages/expo-web-browser/src/ExpoWebBrowser.web.ts]: `replace` is not part of the spec: https://html.spec.whatwg.org/multipage/window-object.html#dom-open

File: packages/expo-web-browser/src/WebBrowser.types.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+};
+
+export type OpenBrowserParams = {
+  toolbarColor?: string;
+  browserPackage?: string;
+  enableBarCollapsing?: boolean;
+  showTitle?: boolean;
+
+  // Web
+  target?: string;
```
Comment (ide) [inline packages/expo-web-browser/src/WebBrowser.types.ts]:
```suggestion
  windowName?: string;
```

File: packages/expo-web-browser/src/WebBrowser.types.ts [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  features?: string;
```
Comment (ide) [inline packages/expo-web-browser/src/WebBrowser.types.ts]:
```suggestion
  windowFeatures?: string;
```

**Review by ide — APPROVED** [review]

**Comment by ide** [comment]
Be sure to rebuild the JS too.

File: packages/expo-web-browser/src/ExpoWebBrowser.web.ts (hunk 1 above)
Comment (EvanBacon) [reply packages/expo-web-browser/src/ExpoWebBrowser.web.ts]: yas it is (maybe): https://www.w3schools.com/jsref/met_win_open.asp

Comment (ide) [reply packages/expo-web-browser/src/ExpoWebBrowser.web.ts]: w3school is not the spec, there is location.replace for that: https://html.spec.whatwg.org/multipage/history.html#dom-location-replace

Comment (EvanBacon) [reply packages/expo-web-browser/src/ExpoWebBrowser.web.ts]: I just copied over the TS definition so it surfaced all available props. Will remove.

**Pull request merged**